router/delivery/http/router_handler.go

```diff
@@ -346,6 +346,21 @@ func getValidRoutingParameters(c echo.Context) (string, sdk.Coin, error) {
 	return tokenOutStr, tokenIn, nil
 }
 
+// getDirectCustomQuoteParameters returns the pool IDs, tokenIn and tokenOutDenom from server context if they are valid.
+func getDirectCustomQuoteParameters(c echo.Context) ([]uint64, []string, sdk.Coin, error) {
+	poolID, tokenOutStr, tokenInStr, err := getPoolsValidTokenInTokensOut(c)
+	if err != nil {
+		return nil, nil, sdk.Coin{}, err
+	}
```

Comment:
Why do we need this new function?

Could we use a combination of the old`getValidRoutingParameters` + the new `getValidPoolID`?